CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing
+
+1. **Please sign one of the contributor license agreements below!**
+1. Fork the repo, develop and test your code changes, add docs.
+1. Make sure that your commit messages clearly describe the changes.
+1. Send a pull request.
+
+## Opening an issue
+
+If you've tried everything in our [Cloud SQL Proxy Client
+docs](https://cloud.google.com/sql/docs/mysql/sql-proxy) and believe you have
+found a bug in the proxy code or an inaccuracy in the documentation, please open
+an issue. GitHub provides a guide, [Mastering
+Issues](https://guides.github.com/features/issues/), that is useful if you are
+unfamiliar with the process. Here are the specific steps for opening an issue:
+
+1. Go to the project issues page on GitHub.
+1. Click the green `New Issue` button located in the upper right corner.
+1. In the title field, write a single phrase that identifies your issue.
+1. In the main editor, describe your issue.
+1. Click the submit button.
+
+Thank you. We will attempt to categorize your issues with an estimate of the
+priority and issue type. We will do our best to comment on your issue within one
+business day. If we believe the issue is a genuine bug, we will try to respond
```

Comment:
Let's remove `If we believe the issue is a genuine bug,` -- It doesn't really address feature requests or documentation questions. We should use the closing of an issue to signal we aren't devoting any time to it.